dotnet/samples/KernelSyntaxExamples/Example55_TextChunker.cs

```diff
@@ -68,10 +67,8 @@ private static void WriteParagraphsToConsole(List<string> paragraphs)
         }
     }
 
-    private static int TokenCounter(string input)
+    private static int CustomTokenCounter(string input)
     {
-        var tokens = GPT3Tokenizer.Encode(input);
-
-        return tokens.Count;
+        return input.Length / 4;
```

Comment:
Isn't it a meaningful gap now that our samples for chunking can't do it based on actual token count?

Reply:
I think it's just a syntax showcase how to use custom token counter. Since `tokenCounter` parameter is optional and can be `null`, in this case the default one will be used, and it has the same logic:
https://github.com/microsoft/semantic-kernel/blob/34456aa1b49486bba0569697116119121bcdfcfb/dotnet/src/SemanticKernel/Text/TextChunker.cs#L281-L284

Reply:
My point is, developers writing these solutions need to be able to compute token counts, whether for chunking or limiting the size of a prompt. What are SK samples for this going to show now?

Reply:
As far as I understand, this PR shouldn't impact already existing SK samples. 
For chunking, users can replace `input.Length / 4` logic with their own logic or use already existing library. We can add more comments if needed. We can showcase how to use some external tokenizer library, but these examples should be already provided by library itself. 

In terms of limiting the size of a prompt, we don't have such examples at the moment. But again, tokenizer libraries should contain such examples. 

Do you have any recommendations here?

Reply:
@dmytrostruk May be worth using the SharpToken in our examples, Like we did for the Polly reliability scenarios.

Reply:
My idea was to avoid using specific tokenizer library in examples, since there are multiple implementations, but if this is preferred approach, it works for me as well. I've added SharpToken in `TextChunker` example in this commit: https://github.com/microsoft/semantic-kernel/pull/2809/commits/7b1cc1901ebe36ac40e3be76bd6dc966318de5ce
Thanks for feedback!